addons/addon-base-raas/packages/base-raas-rest-api/lib/controllers/environment-sc-controller.js

```diff
@@ -68,6 +68,26 @@ async function configure(context) {
     }),
   );
 
+  // ===============================================================
+  //  POST / (mounted to /api/workspaces/service-catalog)
+  // ===============================================================
+  router.post(
+    '/:id/cidr',
+    wrap(async (req, res) => {
+      const requestContext = res.locals.requestContext;
+      const id = req.params.id;
+      const updateRequest = req.body.cidr;
+
+      const [environmentScCidrService] = await context.service(['environmentScCidrService']);
+      const result = await environmentScCidrService.update(requestContext, {
```

Comment:
Where does AuthN/Z happen?

Reply:
Inside the `environmentScCidrService.update` method